### PR TITLE
Add sensors related firmware files and configs

### PR DESCRIPTION
**Pull request by Cobeta-beta**
Yep

Signed-off-by: Victoria Matyukhina <cobeta.beta@gmail.com>

### CODE REVIEW

File: etc/qcom/sns_reg_config [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version=1
```
Comment (JIaxyga) [inline etc/qcom/sns_reg_config]: s sns_reg_config/sns_reg.conf

**Comment by JIaxyga** [comment]
@Gelbpunkt

**Comment by Gelbpunkt** [comment]
Please drop the ipa_fws.elf, rebase your commits and then make sure sensors work fine

**Comment by JIaxyga** [comment]
Thanks

**Comment by Cobeta-beta** [comment]
> Please drop the ipa_fws.elf, rebase your commits and then make sure sensors work fine

I'll try doing that today.

**Pull request merged**